### PR TITLE
doc: manual image approval

### PR DESCRIPTION
**Pull request by Shubham9t9**
# Description

### CODE REVIEW

File: docs/user-guide/creating-application/workflow/cd-pipeline.md
Comment (ashokdevtron) [inline docs/user-guide/creating-application/workflow/cd-pipeline.md:89]: Change `manual approval for deployment` to `Manual approval for deployment`

File: docs/user-guide/creating-application/workflow/cd-pipeline.md
Comment (ashokdevtron) [inline docs/user-guide/creating-application/workflow/cd-pipeline.md:85]: Can we put this in s3 instead of our repo?

File: docs/user-guide/creating-application/workflow/cd-pipeline.md
Comment (ashokdevtron) [inline docs/user-guide/creating-application/workflow/cd-pipeline.md:90]: Change this to
Users can specify the number of approvals required for each deployment, where the allowed limit is minimum 1 approval or maximum 6 approval.

File: docs/user-guide/creating-application/workflow/cd-pipeline.md
Comment (ashokdevtron) [inline docs/user-guide/creating-application/workflow/cd-pipeline.md:94]: To enable manual approval for deployment, follow these steps:

File: docs/user-guide/creating-application/workflow/cd-pipeline.md
Comment (ashokdevtron) [inline docs/user-guide/creating-application/workflow/cd-pipeline.md:96]: Click the deployment pipeline for which you want to enable manual approval.

File: docs/user-guide/creating-application/workflow/cd-pipeline.md
Comment (ashokdevtron) [inline docs/user-guide/creating-application/workflow/cd-pipeline.md:97]: Turn on the ‘Manual approval for deployment’ toggle button.

File: docs/user-guide/creating-application/workflow/cd-pipeline.md
Comment (ashokdevtron) [inline docs/user-guide/creating-application/workflow/cd-pipeline.md:124]: If you want to run actions like closure of Jira ticket after the deployment, you can configure such actions in the post-deployment stages.

File: docs/user-guide/deploying-application/triggering-cd.md
Comment (ashokdevtron) [inline docs/user-guide/deploying-application/triggering-cd.md:17]: When no approved image is available or if the image is already deployed, you will not see any image available for deployment upon clicking the `Select Image` button.

File: docs/user-guide/global-configurations/user-access.md
Comment (ashokdevtron) [inline docs/user-guide/global-configurations/user-access.md:18]: **Admin**: User with `admin` privileges can create, edit, delete, and view permitted applications in permitted projects.

File: docs/user-guide/global-configurations/user-access.md
Comment (ashokdevtron) [inline docs/user-guide/global-configurations/user-access.md:21]: **Approver**: Users with `approver` privileges have the authority to approve requests for image deployment. However, the user who built the image and/or requested approval cannot self-approve, even if they have approver or super-admin privileges.

File: docs/user-guide/deploying-application/triggering-cd.md
Comment (ashokdevtron) [inline docs/user-guide/deploying-application/triggering-cd.md:45]: Click the `Approve` button, and then `Approve Request` to grant approval to the request.

**Review by ashokdevtron — CHANGES_REQUESTED** [review]

File: docs/user-guide/creating-application/workflow/cd-pipeline.md
Comment (prakarsh-dt) [reply docs/user-guide/creating-application/workflow/cd-pipeline.md:85]: This doesn't seems resolved. @ashokdevtron

Comment (ashokdevtron) [reply docs/user-guide/creating-application/workflow/cd-pipeline.md]: Now it's done, @prakarsh-dt

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**
